.gitignore

```diff
@@ -258,4 +258,3 @@ paket-files/
 
 # Python Tools for Visual Studio (PTVS)
 __pycache__/
-*.pyc
```

Comment:
Can you re-add this line please?

Reply:
Sorry.  I thought that was something I added because it was highlighted in the DIFF.
re-added.